cfgov/unprocessed/css/organisms/footer.less

```diff
@@ -150,62 +150,63 @@
                 }
 
             }
-        });
+        } );
     }
 
     &-middle-left {
-        .footer_list {
+        .o-footer_list {
             margin: 0;
         }
 
-        .respond-to-min(@bp-sm-min, {
+        .respond-to-min( @bp-sm-min, {
             .grid_column(8);
             border-right: 1px solid @gray-40;
             border-left: 0;
 
-            .footer_col {
+            .o-footer_col {
                 .grid_column(6);
                 border-left: 0;
                 border-right: 0;
+                padding-right: unit( @grid_gutter-width / 2 / @base-font-size-px, em );
```

Comment:
This is to take care of this issue:

![screen shot 2016-04-06 at 1 01 29 pm](https://cloud.githubusercontent.com/assets/704760/14326124/8e7d9968-fbfa-11e5-911d-dd64c7ae1a53.png)